src/libs/address.js

```diff
@@ -1,30 +1,41 @@
 import IdunnPoi from '../adapters/poi/idunn_poi';
 
 /**
- * Format an address given an address object (name, city, country, label, and other regions)
+ * Filter an address and return an array with the relevant items
  * @param {*} address - an address object
  */
-export function format(address) {
-  if (!address) {return '';}
+export function filter(address) {
+
+  if (!address) {return [];}
+
+  const city = address.postcode && address.city
+    ? address.postcode + ' ' + address.city
+    : address.city;
 
   if (!address.street) {
     return [
       address.suburb,
       address.cityDistrict,
-      address.city,
+      city,
```

Comment:
I would not use the postcode here, as this format is used to provide a context about where a point or zone is located if no postal address is appropriate.